packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Checkbox from './Checkbox';
+import { applyTheme } from 'cf-style-container';
 import includes from 'lodash/includes';
 
+import CheckboxUnstyled from './Checkbox';
+import CheckboxTheme from './CheckboxTheme';
+
+const Checkbox = applyTheme(CheckboxUnstyled, CheckboxTheme, theme => ({
```

Comment:
There's nothing I'd want to add right now but if we follow this path I can see us having: `inAbox`, `inACardSection`, `inACardBlock`, `inAPopUpModal`, `inInput`, etc. rather than putting the styles that are needed for each implementation by using `applyTheme` to override the styles. The latter seems more reasonable to me and it's kind of the path I've been going down with a lot of the components I've migrated.

Maybe I should just pull the `<Checkbox />` components out like I did in the radio migration (#333) and you suggested below. It makes a lot of these issues disappear.